### ISSUE TITLE
Special characters in demangled names replaced with underscores

### ISSUE THREAD

**Issue by io12** [radareorg/radare2#10891]
I realize that this is a deliberate design decision because those characters have a special meaning in the command language, but it can make complex types substantially less readable. Maybe this could be an option and the autocompletion could automatically escape the *problem* characters?

**Comment by io12** [comment]
Yes, but realname still filters out spaces, and the filtered flag names are displayed in the disassembly.

**Comment by radare** [comment]
it shouldnt filter the realname. can you provide a reproducer?

**Comment by io12** [comment]
From what I can see, this happens for all mangled flags. https://github.com/radare/radare2/blob/master/libr/bin/demangle.c#L154

**Comment by ret2libc** [comment]
Can we define what this issue is really about before? Is it about making sure realnames are not escaped? Or about removing the realname/flagname at all? Or just adding a config var? Or what? Issues without a clear way forward are easily left behind and never acted upon. @XVilka
